bootstrap/stuff.py

```diff
@@ -0,0 +1,14 @@
+from helpers import *
+
+r = redis()
+
+users = get_user_list_from_redis(r)
+user_data = pull_user_data_from_redis(users, r)
+
+data = {}
+
+for piece in user_data:
+    data[piece.get_username()] = piece.to_dict()
+
+with open("redis_data.json", "w") as outfile:
+    json.dump(data, outfile, indent=2)
```

Comment:
What is this for?

Reply:
generates the redis_data.json file that we were using. Can be removed.